app/CMakeLists.txt

```diff
@@ -15,4 +15,12 @@ if (APPLE)
     target_sources(DeltaPatcher PRIVATE ${bundle_icon})
     set_source_files_properties(${bundle_icon} PROPERTIES MACOSX_PACKAGE_LOCATION Resources/)
     set_target_properties(DeltaPatcher PROPERTIES MACOSX_BUNDLE TRUE MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/mac_stuff/Info.plist)
-endif()
\ No newline at end of file
+endif()
+
+install(TARGETS DeltaPatcher DESTINATION bin)
```

Comment:
Since DeltaPatcher is a single binary, we can avoid the complex install(TARGETS ..., and instead use the install(FILES and simply copy the binary. The only real dependencies Delta Patcher has, when built in Release mode and with wxBUILD_SHARED=OFF is the system's libc and gtk3, which are already installed, since cmake was able to compile it, the rest is statically linked.

Reply:
I would rather keep the install target. This makes sure everything, including the data files, is installed in the right place. This is common on Linux, even with single binary programs.

Reply:
My concern is that it will install the x binary, which could overwrite the system one, cause conflicts

Reply:
If I don't install the dependencies DeltaPatcher will show this error: `libwx_gtk3u_core-3.3.so.0`, so it's not statically compiled. I will add a Option for installing the decencies.

Reply:
Even using the wxBUILD_SHARED=OFF flag?

Reply:
I totally forgot to set the Option. Sorry,

Reply:
cmake -B build -DCMAKE_BUILD_TYPE=Release -DwxBUILD_SHARED=OFF